docs/tendermint-core/consensus/proposer-based-timestamps.md

```diff
@@ -0,0 +1,89 @@
+---
+order: 3
+---
+
+# Proposer-Based Timestamps
+
+ This document provides an overview of the Proposer-Based Timestamp algorithm 
+ added to Tendermint in the v0.36 release. This document outlines the core functionality
+ as well as the parameters and constraints of the this algorithm.
+
+# Algorithm Overview 
+
+The proposer-based timestamps algorithm defines a way for a Tendermint blockchain
+to create block timestamps that are within a reasonable bound of the clocks of the
+validators on the network.
+
+## Algorithm Parameters
+
+The functionality of the Proposer-Based Timestamps algorithm is governed by two
+parameters within Tendermint. These two parameters are [consensus parameters](https://github.com/tendermint/tendermint/blob/c42c6d06d2986791ad1cb8742824818edd9b6323/spec/abci/apps.md#L291), 
+meaning they are configured by the ABCI application and are expected to be the
+same across all nodes on the network.
+
+### `Precision`
+
+The `Precision` parameter configures the acceptable upper-bound of clock drift among
+all of the nodes on a Tendermint network. Any two nodes on a Tendermint network
+are expected to have clocks that differ by at most `Precision` seconds? Ms? at
+any given instant.
+
+### `MessageDelay`
+
+The `MessageDelay` parameter configures the acceptable upper-bound for transmitting
+a `Proposal` message from the proposer to _all_ of the validators on the network.
+
+Networks that want very tight bounds on their timestamps should set a smaller value
+for `MessageDelay`. Highly distributed networks or networks with generally slow
+rates of transmission should set a larger value for `MessageDelay`.
```

Comment:
As @tychoish would say, it's nice to want things 🙂. Perhaps we should frame this so the reader can see a rubric, e.g., "Networks should choose as small a value of `MessageDelay` as is practical, provided it is large enough that messages can reach all participants with high probability given the size and latency of their connections."

The wording here probably needs work, but I hope that illustrates the idea: Not just why to increase it or decrease it, but what the mutual constraints are.